src/components/Button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Button({ clickHandler, children, type, disabled }) {
+    return (
+        <button
+            onClick={clickHandler}
+            type={type}
+            disabled={disabled || false}
+        >
+            {children}
+        </button>
+    );
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken